bincrafters_envy/bincrafters_envy.py

```diff
@@ -1,308 +1,115 @@
-                host=travis_host,
-                accountName=travis_account,
-                projectSlug=project_slug,
-                id=ids[name]
-            )
-            r = requests.patch(travis_url_env, data=json.dumps(request), headers=travis_headers)
-            if r.status_code != 200:
-                raise Exception('travis PATCH request failed %s %s' % (r.status_code, r.content))
-        else:
-            r = requests.post(travis_url, data=json.dumps(request), headers=travis_headers)
-            if r.status_code != 201:
-                raise Exception('travis POST request failed %s %s' % (r.status_code, r.content))
+try:
+    from configparser import ConfigParser
+except ImportError:
+    from ConfigParser import ConfigParser
 
+from .appveyor import Appveyor
+from .travis import Travis
+from .circle import Circle
+from .azure import Azure
 
-def appveyor_encrypt(value):
-    appveyor_url = '{host}/api/account/encrypt'.format(host=appveyor_host)
-    request = dict()
-    request['plainValue'] = value
-    r = requests.post(appveyor_url, data=json.dumps(request), headers=appveyor_headers)
-    if r.status_code != 200:
-        raise Exception('appveyor POST request failed %s %s' % (r.status_code, r.content))
-    return r.content.decode()
-
-
-def update_appveyor(project_slug, env_vars, encrypted_vars):
-    appveyor_url = '{host}/api/projects/{accountName}/{projectSlug}/settings/environment-variables'.format(
-        host=appveyor_host,
-        accountName=appveyor_account,
-        projectSlug=project_slug.replace('_', '-')
-    )
-
-    r = requests.get(appveyor_url, headers=appveyor_headers)
-    if r.status_code != 200:
-        raise Exception('appveyor GET request failed %s %s' % (r.status_code, r.content))
-    appveyor_vars = json.loads(r.content.decode())
-
-    new_env_vars = dict()
-    for k, v in env_vars.items():
-        new_var = dict()
-        new_var['name'] = k
-        new_var['value'] = dict()
-        if k in encrypted_vars:
-            new_var['value']['value'] = appveyor_encrypt(v)
-            new_var['value']['isEncrypted'] = True
-        else:
-            new_var['value']['value'] = v
-            new_var['value']['isEncrypted'] = False
-        new_env_vars[k] = new_var
-
-    for v in appveyor_vars:
-        name = v['name']
-        if name not in new_env_vars.keys():
-            new_env_vars[name] = v
-
-    request = []
-
-    for _, v in new_env_vars.items():
-        request.append(v)
-
-    r = requests.put(appveyor_url, data=json.dumps(request), headers=appveyor_headers)
-    if r.status_code != 204:
-        raise Exception('appveyor PUT request failed %s %s' % (r.status_code, r.content))
-
-
-def yes_no():
-    print('[y/n]')
-    choice = input().lower()
-    while choice not in ['y', 'n']:
-        print('please respond with y or n')
-        choice = raw_input().lower()
-    return choice == 'y'
-
+__author__ = "BinCrafters"
+__license__ = "MIT"
+__version__ = '0.1.7'

```

Comment:
```suggestion
__version__ = '0.2.0'
```
Big change requires a new minor ;)